prow/e2e-suite.sh

```diff
@@ -27,10 +27,15 @@ set -u
 set -x
 
 if [ "${CI:-}" == 'bootstrap' ]; then
+  # Make sure we are in the right directory
   # Test harness will checkout code to directory $GOPATH/src/github.com/istio/istio
   # but we depend on being at path $GOPATH/src/istio.io/istio for imports
-  ln -sf ${GOPATH}/src/github.com/istio ${GOPATH}/src/istio.io
-  cd ${GOPATH}/src/istio.io/istio
+  if [[ ! $PWD = ${GOPATH}/src/istio.io/istio ]]; then
+    # Test harness will checkout code to directory $GOPATH/src/github.com/istio/istio
```

Comment:
shouldn't that be configurable ? it's pretty standard to have foo.io instead of github.com - it could read the CNAME for instance

Reply:
Can we do something smarter later maybe and use this for now :)